apps/src/templates/projects/projectsModule.js

```diff
@@ -26,4 +77,24 @@ export function selectGallery(projectType) {
   return { type: TOGGLE_GALLERY, projectType: projectType };
 }
 
-registerReducers({selectedGallery});
+/**
+ * Takes a list of older projects and appends it to the existing list of
+ * projects of the specified type.
+ * @param {Array} projects A list of projects which are all older than the
+ * the current oldest project, newest first.
+ * @param {string} projectType The type of the projects being added.
+ * @returns {{projects: Array, projectType: string, type: string}}
+ */
+export function addOlderProjects(projects, projectType) {
+  return {projects, projectType, type: ADD_OLDER_PROJECTS};
+}
+
+export function setProjectLists(projectLists) {
+  return {projectLists, type: SET_PROJECT_LISTS};
+}
+
+export function setHasOlderProjects(hasOlderProjects, projectType) {
+  return {hasOlderProjects, projectType, type: SET_HAS_OLDER_PROJECTS};
+}
```

Comment:
That seems like a good convention. I'll do that despite the [style rule](https://github.com/airbnb/javascript#objects--grouped-shorthand) which says otherwise